### PR TITLE
Update storage-replica-frequently-asked-questions.yml

### PR DESCRIPTION
**Pull request by JohnMarlin-MSFT**
Added additional question and answer.

### CODE REVIEW

**Comment by PRMerger15** [comment]
@JohnMarlin-MSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by v-dasis** [comment]
#sign-off

**Comment by PRMerger9** [comment]
@v-dasis: I'm sorry - only the author of this article, @nedpyle, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

File: WindowsServerDocs/storage/storage-replica/storage-replica-frequently-asked-questions.yml [hunk 1]
```diff
@@ -266,7 +266,12 @@ sections:
           
           - Server1 source drive M: with source log drive L: replicating to Server2 destination drive M: with destination log drive L:
           - Server2 source drive O: with source log drive N: replicating to Server1 destination drive O: with destination log drive N:
-          
+
+      - question: |
+          Can you place cluster disks in maintenance mode?
+        answer: |
+          Storage Replica will block any cluster disks from entering maintenance mode.  For tasks such as enabling or disabling Bitlocker, the disks must be in maintenance mode.  In order to accomplish these type tasks, the partnership would need to be broken first and created again once it is complete.
```
Comment (JasonGerend) [inline WindowsServerDocs/storage/storage-replica/storage-replica-frequently-asked-questions.yml:273]:
```suggestion
          Storage Replica will block any cluster disks from entering maintenance mode.  For tasks such as enabling or disabling Bitlocker, the disks must be in maintenance mode.  To perform tasks that require the disks to be in maintenance mode, the partnership would need to be broken first and created again once it is complete.
```

**Comment by PRMerger18** [comment]
@JasonGerend : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by JasonGerend — APPROVED** [review]

**Review by nedpyle — COMMENTED** [review]
Looks good to me

**Pull request merged**